script/testing/oltpbench/configs/end_to_end_performance/tpcc.json

```diff
@@ -1,5 +1,10 @@
 {
     "type": "oltpbenchmark",
+    "server_args":{
+        "wal": false,
+        "connection_thread_count": 32,
+        "log_file_path": "/mnt/ramdisk/wal.log"
```

Comment:
If `wal == false` I wouldn't think we need to specify a log_file_path

Reply:
I think you are right. I think there is a requirement conflict between https://github.com/cmu-db/terrier/issues/1062 and https://github.com/cmu-db/terrier/issues/1054

Reply:
Yah I think there is one test with `wal:false` one with RAM disk and one with neither, but we should get Andy to confirm